examples_gbs/run_tutorial_similarity.py

```diff
@@ -0,0 +1,278 @@
+# pylint: disable=wrong-import-position,wrong-import-order,ungrouped-imports,invalid-name
+"""
+Graph Similarity Tutorial
+=========================
+
+This tutorial looks at how to use GBS to construct a similarity measure between graphs,
+what is known as a graph kernel :cite:`schuld2019quantum`. Kernels can be applied to graph-based
+data for machine learning tasks such as classification in machine learning using a support vector
+machine.
+
+Graph data
+----------
+
+Let's use the MUTAG dataset of graphs for this tutorial
+:cite:`debnath1991structure,kriege2012subgraph`. This is a dataset of 188 different graphs that
+each correspond to the structure of a chemical compound. Our objective is to use GBS samples from
+these graphs to measure their similarity.
+
+The :mod:`~.gbs.data` module provides pre-calculated GBS samples from four graphs in the
+MUTAG dataset. We'll start by loading these sample sets and visualizing the corresponding graphs.
+"""
+
+from strawberryfields.gbs import data, plot, similarity
+
+m0 = data.Mutag0()
+m1 = data.Mutag1()
+m2 = data.Mutag2()
+m3 = data.Mutag3()
+
+##############################################################################
+# These datasets contain both the adjacency matrix of the graph and the samples generated through
+# GBS. We can access the adjacency matrix through:
+
+m0_a = m0.adj
+m1_a = m1.adj
+m2_a = m2.adj
+m3_a = m3.adj
+
+##############################################################################
+# We can now plot the four graphs using the :mod:`~gbs.plot` module. To use this module,
+# we need to convert the adjacency matrices into NetworkX Graphs:
+
+import networkx as nx

```

Comment:
Reminder that we should discuss and agree on an import structure for the tutorials